code/build.R

```diff
@@ -85,9 +87,27 @@ m1 = r_main_region %>%
 # + tm_text("OrganisationName")
 # m1
 
+##Update cycling potential with new PCT figures
+rnet = pct::get_pct_rnet(region = "avon")
+rnet_buff = geo_buffer(shp = rnet, dist = 10)
+r_cyipt_joined = st_join(r_main_region, rnet_buff, join = st_within)
+
+dupes = r_cyipt_joined[which(duplicated(r_cyipt_joined$idGlobal) == TRUE | duplicated(r_cyipt_joined$idGlobal, fromLast = TRUE) == TRUE),]
+dupes_max = dupes %>% 
+  st_drop_geometry() %>% 
+  group_by(idGlobal) %>%
+  summarise(cycling_potential_max = max(govtarget_slc)) 
+
+r_joined = left_join(r_cyipt_joined, dupes_max, by = "idGlobal") %>%
+  mutate(cycling_potential = ifelse(is.na(cycling_potential_max), ifelse(is.na(govnearmkt_slc), pctgov, govnearmkt_slc), cycling_potential_max),
```

Comment:
If the duplicated match is NA and updated PCT result is NA take the origin, else take the largest matching value. Makes sense.